.circleci/config.yml

```diff
@@ -76,11 +66,13 @@ jobs:
       - run:
           name: Update Carthage
           command: brew outdated carthage || (brew uninstall carthage --force; HOMEBREW_NO_AUTO_UPDATE=1 brew install carthage --force-bottle)
-      - run: carthage build --no-skip-current && for platform in Mac iOS tvOS; do test -d Carthage/Build/${platform}/Anchorage.framework || exit 1; done
+      - run: carthage build --no-skip-current --configuration .circleci/SWIFT_5.0.xcconfig && for platform in Mac iOS tvOS; do test -d Carthage/Build/${platform}/Anchorage.framework || exit 1; done
 
-  carthage-build-xcode-10:
+  carthage-build-xcode-10.1:
     macos:
-      xcode: "10.0.0"
+      xcode: "10.1.0"
+    environment:
+      XCODE_XCCONFIG_FILE: ./.circleci/SWIFT_4.2.xcconfig
```

Comment:
Job command is missing `XCODE_XCCONFIG_FILE`

Reply:
Oops, I think I did this backwards. I believe we can delete the env variable now that I’m passing the value directly. I’ll delete it from here and confirm that it still works. `XCODE_XCCONFIG_FILE` is an env variable defined in `man xcodebuild`.